gpu_bdb/cluster_configuration/cluster-startup.sh

```diff
@@ -1,27 +1,50 @@
-ROLE=$1
+#!/bin/bash
+
+#########################################################
+### Configuration to (possibly) tweak ###
+#########################################################
 USERNAME=$(whoami)
 
-# NVLINK or TCP, based on environment variable (or manual setting)
-if [ -z "$CLUSTER_MODE" ]
-then
-    CLUSTER_MODE="TCP"
-else
-    CLUSTER_MODE=$CLUSTER_MODE
-fi
+# Scheduler configuration for your machine or cluster
+SCHEDULER_PORT=${SCHEDULER_PORT:-8786}
+DASHBOARD_ADDRESS=${DASHBOARD_ADDRESS:-8787}
+INTERFACE=${INTERFACE:-ib0}
+
+# Logging and scratch space for your machine or cluster
+LOCAL_DIRECTORY=${LOCAL_DIRECTORY:-/raid/$USERNAME/dask-local-directory}
+SCHEDULER_FILE=${SCHEDULER_FILE:-$LOCAL_DIRECTORY/scheduler.json}
+LOGDIR=${LOGDIR:-$LOCAL_DIRECTORY/logs}
+WORKER_DIR=${WORKER_DIR:-/raid/$USERNAME/gpu-bdb-dask-workers/}
+
+# Communication protocol
+CLUSTER_MODE=${CLUSTER_MODE:-TCP}
 
+# Cluster memory configuration
 MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
-DEVICE_MEMORY_LIMIT="18GB"
-POOL_SIZE="29GB"
+DEVICE_MEMORY_LIMIT=${DEVICE_MEMORY_LIMIT:-15GB}
+POOL_SIZE=${POOL_SIZE:-29GB}
 
-GPU_BDB_HOME=$HOME/gpu-bdb
-CONDA_ENV_NAME="rapids-gpu-bdb"
-CONDA_ENV_PATH="/home/$USERNAME/conda/etc/profile.d/conda.sh"
+# Conda environment information
+CONDA_ENV_NAME=${CONDA_ENV_NAME:-rapids-gpu-bdb}
+CONDA_ENV_PATH=${CONDA_ENV_PATH:-/raid/$USERNAME/miniconda3/etc/profile.d/conda.sh}
 
-# Used for writing scheduler file and logs to shared storage
-LOCAL_DIRECTORY=$HOME/dask-local-directory
-SCHEDULER_FILE=$LOCAL_DIRECTORY/scheduler.json
-LOGDIR="$LOCAL_DIRECTORY/logs"
-WORKER_DIR=/tmp/$USERNAME/gpu-bdb-dask-workers/
+# Repository home
+GPU_BDB_HOME=${GPU_BDB_HOME:-/raid/$USERNAME/prod/gpu-bdb}
+
+# Dask-cuda optional configuration
+export DASK_JIT_UNSPILL=${DASK_JIT_UNSPILL:-True}
```

Comment:
Just confirming that this is intentional.

Reply:
Yes, turning this on and leaving BlazingSQL off as the default